### PR TITLE
RYA-155 Accumulo related config properties are separated into an extended config object.

### PR DESCRIPTION
**Pull request by ejwhite922**

### CODE REVIEW

**Comment by isper3at** [comment]
A lot of the fields that shouldn't be in the MergeConfiguration still made it in in your last PR.  Please remove them in this commit and put them in the Accumulo only config

**Comment by isper3at** [comment]
I'm also going to clean up the CLI driver so the accumulo/mongo stuff are in their own place to make it look a lot cleaner

File: extras/rya.export/export.api/src/main/java/org/apache/rya/export/api/conf/MergeConfiguration.java [hunk 1]
```diff
@@ -67,67 +61,35 @@
...
+        } catch(final NullPointerException npe) {
+            throw new MergeConfigurationException("The configuration was passed a null builder.", npe);
+        }
+        try {
+            this.parentHostname = checkNotNull(builder.parentHostname);
+            this.parentUsername = checkNotNull(builder.parentUsername);
+            this.parentPassword = checkNotNull(builder.parentPassword);
+            this.parentRyaInstanceName = checkNotNull(builder.parentRyaInstanceName);
+            this.parentTablePrefix = checkNotNull(builder.parentTablePrefix);
+            this.parentTomcatUrl = checkNotNull(builder.parentTomcatUrl);
```
Comment (isper3at) [inline extras/rya.export/export.api/src/main/java/org/apache/rya/export/api/conf/MergeConfiguration.java:78]: I don't think we need the tomcat url here either

Comment (ejwhite922) [inline extras/rya.export/export.api/src/main/java/org/apache/rya/export/api/conf/MergeConfiguration.java:78]: It's needed for time synchronization.  When time sync'ing is used, an HTTP GET request is sent to a web server running on the remote machine (in this case Tomcat running on the parent or child machine) to determine the remote's local time.

Comment (isper3at) [inline extras/rya.export/export.api/src/main/java/org/apache/rya/export/api/conf/MergeConfiguration.java:78]: gotcha, sounds good. everything else good to go?  This looks good to pull in

**Pull request merged**